src/database.rs

```diff
@@ -153,6 +152,18 @@ pub struct DbOptions {
     pub jobs: usize,
 }
 
+impl Default for DbOptions {
+    fn default() -> Self {
+        Self {
+            autosplit: false,
+            timing: false,
+            trace_recalc: false,
+            incremental: false,
```

Comment:
In the grammar and formula tests, and for any work with the grammar parser, I have to set incremental to `true`. This is because [only then statement labels are allocated atoms](https://github.com/david-a-wheeler/metamath-knife/blob/df289140ac1076b0cd0cd8514802cf847024c620/src/nameck.rs#L62), and I use those atoms as a very lightweight statement identifier in formulas.

Therefore, I'd propose the incremental option to be set by default. But it's not a must of course.

Reply:
The basis for the defaults here is what you would get if you ran the CLI with no arguments. Should we reverse the polarity of the incremental setting in the options too?

Reply:
I think the default shall be "create atoms for statement labels": to be honest, I don't see the link with "incremental" build.

So maybe what I'm asking for here is actually either to always create atoms for statement labels, or to split this flag into two: one for creating atoms for statement labels, and one for actual incremental parsing, since "create atoms for statement labels" now has two uses: one for incremental parsing as before, and one for grammar parsing.

Let's just keep this as is for now and handle this in another issue/PR. Incremental build is another story.

Reply:
Created issue #39 to track this.